keyboards/handwired/carpolly/readme.md

```diff
@@ -0,0 +1,15 @@
+# carpolly
+
+*Keyboard Maintainer: Paul Enrico N. Viola (https://github.com/PollyV1)
+*Hardware Supported: Teensy 2 controller
+*Hardware Availability: https://github.com/PollyV1/Carpolly
+
+carpolly - a handwired keyboard on a teensy 2 controller
+
+https://imgur.com/w691sNn
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```